<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentrerTables</class>
 <widget class="QDialog" name="rentrerTables">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rentrer les tables</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>150</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre total de tables</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbTables">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouterTableButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter une table</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gererTypesTablesButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>181</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>GÃ©rer les types de tables</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>561</width>
     <height>351</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>524</width>
      <height>349</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="msgErreur">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
